lib/src/components/NxSubmitMask/NxSubmitMask.tsx

```diff
@@ -23,14 +25,19 @@ const NxSubmitMask: FunctionComponent<Props> =
           'nx-submit-mask--success': !!success
         }),
 
-        // the default Loading... message is provided by NxLoadingSpinner
-        spinnerMessage = success ? (successMessage || 'Success!') : message;
+        showMessage = success ? (successMessage || 'Success!') : message;
 
     return (
       <div className={classes}>
-        <h3 className="nx-h3 nx-submit-mask__message">
-          <NxLoadingSpinner>{spinnerMessage}</NxLoadingSpinner>
-        </h3>
+        <div className="nx-submit-mask__message">
+          { success ?
+            <>
+              <NxFontAwesomeIcon icon={faCheckCircle} className="nx-loading-spinner__icon" />
+              <span className="nx-submit-mask__message-text">{showMessage}</span>
+            </> :
+            <NxLoadingSpinner>{showMessage || 'Submitting...'}</NxLoadingSpinner>
```

Comment:
So we are changing the default text here?

Reply:
> So we are changing the default text here?

Yes, I think it's odd for the default task of the Submit Mask to be "Loading...", it isn't loading, it's submitting.